HTML - 20/changeToCyrillic.js

```diff
@@ -0,0 +1,59 @@
+let latinLetters = document.getElementById("latin").value.toString().toLowerCase();
+
+function changing() {
+    document.getElementById("cyr").innerHTML = getCyrillic(latinLetters).toString();
+}
+
+function getCyrillic(latin) {
+    let cyrillic = latin;
+    let latinString = latin.split('');
+    let cyrillicString = cyrillic.split('');
+    for (let i = 0; i < latinString.length; i++) {
+        if (latinString[i] === "l" && latinString[i+1] === "j") {
+            cyrillicString[i] = "љ";

```

Comment:
https://stackoverflow.com/questions/1431094/how-do-i-replace-a-character-at-a-particular-index-in-javascript

Because you can't do this:`cyrillicString[i] = "љ";` you can use replacement function suggested on that link

```
function setCharAt(str,index,chr) {
    if(index > str.length-1) return str;
    return str.substr(0,index) + chr + str.substr(index+1);
}
```
And this code will be:  `cyrillicString=setCharAt(cyrillicString,i ,"љ");`